src/test/java/dk/sdu/se_f22/productmodule/management/JSONReaderTest.java

```diff
@@ -0,0 +1,47 @@
+package dk.sdu.se_f22.productmodule.management;
+
+import dk.sdu.se_f22.sharedlibrary.models.Product;
+import org.junit.jupiter.api.Test;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class JSONReaderTest {
+
+    private static JSONReader reader = new JSONReader("src/test/resources/dk/sdu/se_f22/productmodule/management/products.json");
+
+    public void runAllTests(){
+        read();
+        write();
+    }
+
+    @Test
+    void read() {
+
+        ArrayList<Product> products = null;
+        try {
+            assertFalse(reader.read().isEmpty());          //Is there any Products in the array?
+            assertNotNull(products = reader.read());       //Does the array exist?
+            assertTrue(reader.read().size() > 1);  //Is there more than 1 product object?
```

Comment:
These are good, however I don't think it makes sense to re-read the entire file three times, when not modifying it.
It could just be read once, and then have those 3 asserts still.